assignments/99_00_sql/99_02_tokenizer/tokenizer/stokenize.h

```diff
@@ -43,4 +43,8 @@ class STokenizer
   
 };
 
+// initializing static member
+// see: https://en.cppreference.com/w/cpp/language/static
+int STokenizer::_table[MAX_ROWS][MAX_COLUMNS];
```

Comment:
As your comments, should this line put in `.ccp` file instead of `.h` file?
```cpp
#include "stokenize.h"
int STokenizer::_table[MAX_ROWS][MAX_COLUMNS];
```

Reply:
put it in a new stokenize.cpp